src/main/java/io/github/perrymant/moneymaker/Application.java

```diff
@@ -18,6 +24,8 @@ void start(String[] args) {
                 logger.info(reader.read("moneymaker-help.txt"));
             } else if ("report".equals(args[0])) {
                 logger.info(budget.report());
+            } else if ("reportJSON".equals(args[0])) {
```

Comment:
I don't understand what this is. Let's stick to the engineering design and task breakdown unless there's a really good reason not to.

Just to clarify: I'd expect #34 to result in a PR with literally only one file, and #35 to *replace* `report`, not add another.